build/Makefile.objecthash

```diff
@@ -0,0 +1,4 @@
+all: test
+
+test:
+	$(MAKE) c CPPFLAGS="-I $(INSTALL_DIR)/include" LDFLAGS="-L $(INSTALL_DIR)/lib"
```

Comment:
Is it necessary to run the dep tests? Strictly, I guess not, but since it's quick...